.travis.yml

```diff
@@ -39,21 +39,6 @@ jobs:
         - gem install brakeman
         - brakeman --run-all-checks
 
-  allow_failures:
-
-    - stage: test
-      name: 'bundle audit'
-      script:
-        - gem install bundler-audit
-        - bundle audit update
-        - bundle audit check
-
-    - stage: test
-      name: 'brakeman check'
-      script:
-        - gem install brakeman
-        - brakeman --run-all-checks
-
```

Comment:
no likey?

Reply:
Confusingly, this is just under the `allow_failures` key, for some reason you need to duplicate them so they still run, they just aren't allowed to fail anymore